### DISCUSSION TITLE
is it possible to use it with llama.cpp

### DISCUSSION THREAD

**Discussion by kroryan** [neph1/LlamaTale#35] [General]
so i would like to know if is possible to use it with llama.cpp using ./server and listening port 5001, i tried and didnt work

**Comment by neph1** [comment]
It should be, it's just not configured.
Try sticking to the koboldcpp config, but change "ENDPOINT" to "/completion/"
https://github.com/ggerganov/llama.cpp/blob/master/examples/server/README.md

**Reply by kroryan** [reply]
so i have to open the llm_config.yaml and change "/api/v1/generate" to "/v1/chat/completions" ? or where do you mean to change it?

**Reply by neph1** [reply]
Yes, but just to "completion" according to that doc.

**Reply by kroryan** [reply]
I wrote /completion as endpoint and didnt work i will try only with completion

**Reply by neph1** [reply]
I'm not familiar with llama.cpp, I'm just guessing from the documentation. But I see no reason why it shouldn't work.

**Comment by neph1** [comment]
Not sure if you solved this, but I just added a guide to the wiki: https://github.com/neph1/LlamaTale/wiki/Configuring-llama.cpp-server
